ledger/src/blockstore.rs

```diff
@@ -1854,6 +1866,61 @@ impl Blockstore {
         Ok(i)
     }
 
+    fn purge_special_columns_exact(
```

Comment:
hehe this module is becoming terribly large and unwieldy isn't it? Would you mind moving the purge logic/tests to a submodule under `blockstore`?